pkg/api/instances.go

```diff
@@ -54,6 +55,10 @@ type Instance struct {
 	CreatedTs   time.Time           `db:"created_ts" json:"created_ts"`
 	Application InstanceApplication `db:"application" json:"application,omitempty"`
 }
+type InstanceWithTotal struct {
```

Comment:
@joaquimrocha suggested to use `InstancesWithTotal` which makes sense.
 
```suggestion
type InstancesWithTotal struct {
```